backend/config/config.go

```diff
@@ -217,6 +218,7 @@ func NewConfig(appConfigFilename string, accountsConfigFilename string) *Config
 		accountsConfig:         newDefaultAccountsonfig(),
 	}
 	config.load()
+	config.appConfig, _ = config.checkAPIKeys(config.appConfig)
```

Comment:
never ignore errors (rare exception when handling it is impossible or you know for a fact the error can't happen, but then it should be turned into a panic)

Reply:
Good catch, fixed.